### PR TITLE
Fixes #25968: Add a tip message after promote a node to relay to trigger agent run

### PR DESCRIPTION
**Pull request by ElaadF**
https://issues.rudder.io/issues/25968

### CODE REVIEW

File: scale-out-relay/src/main/resources/server-node-to-relay [hunk 1]
```diff
@@ -63,5 +63,6 @@ else
 	echo " Done."
 
 	echo "Success: all steps completed successfully."
+	echo "TIP: on the relay you can trigger 'rudder agent run' to download initial policies immediatly.
```
Comment (amousset) [inline scale-out-relay/src/main/resources/server-node-to-relay:66]:
```suggestion
	echo "TIP: on the relay you can trigger 'rudder agent run' to download updated policies immediatly.
```

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Comment by ElaadF** [comment]
Commit modified

**Pull request merged**